spec/helpers/application_helper_spec.rb

```diff
@@ -5,19 +5,17 @@
 
   describe '#link_to_current_sha' do
```

Comment:
This method no longer exists in the system. The method that you have made no longer returns a link as this is done in the view?